src/web/client/extension.ts

```diff
@@ -601,20 +598,24 @@ function isActiveDocument(fileFsPath: string): boolean {
     );
 }
 
+async function fetchArtemisData(orgId: string) : Promise<string> {
+        const artemisResponse = await fetchArtemisResponse(orgId, WebExtensionContext.telemetry.getTelemetryReporter());
+        if (!artemisResponse) {
+            // Todo: Log in error telemetry. Runtime maintains another table for this kind of failure. We should do the same.
```

Comment:
This is a good candidate for logging in a default/fallback cluster let's say 'US'. Here the only log would be unable to instantiate telemetry because of lack of org info.

Reply:
This geo not only help us to determine the org geo but also initialize 1DS SDK. I would not use 'US' in this case because post this call all data will start flowing to US cluster. There is a table Runtime uses for all such cases will leverage the same.